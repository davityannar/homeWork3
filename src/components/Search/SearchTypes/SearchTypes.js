import React from 'react';
import Types from './Types/Types';

import './SearchTypes.scss';

const SearchTypes = () => {
    return (
        <div className='search-filters'>
            <p>
                Search by
            </p>
            <Types inputId='title' inputClass='filter'  inputName='search' inputType='radio' inputValue='TITLE' inputText='Title' checkedByDefault />
            <Types inputId='genre' inputClass='filter'  inputName='search' inputType='radio' inputValue='GENRE' inputText='Genre' />
        </div>
    )
}
export default SearchTypes;