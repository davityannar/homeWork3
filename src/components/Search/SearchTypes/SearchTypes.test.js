import React from 'react';
import SearchTypes from './SearchTypes';

describe('Tests SearchTypes component', () => {
  it('Matching snapshot', () => {
    const component = shallow(<SearchTypes />)

    expect(component).toMatchSnapshot();
  })
})