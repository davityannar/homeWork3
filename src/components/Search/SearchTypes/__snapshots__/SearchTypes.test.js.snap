// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests SearchTypes component Matching snapshot 1`] = `
<div
  className="search-filters"
>
  <p>
    Search by
  </p>
  <Types
    checkedByDefault={true}
    inputClass="filter"
    inputId="title"
    inputName="search"
    inputText="Title"
    inputType="radio"
    inputValue="TITLE"
  />
  <Types
    inputClass="filter"
    inputId="genre"
    inputName="search"
    inputText="Genre"
    inputType="radio"
    inputValue="GENRE"
  />
</div>
`;
